gw_eccentricity/eccDefinition.py

```diff
@@ -1372,6 +1415,39 @@ def measure_ecc(self, tref_in=None, fref_in=None):
             return_dict.update({"tref_out": self.tref_out})
         return return_dict
 
+    def set_eccentricity_and_mean_anomaly_to_zero(
+            self, tref_in, fref_in):
+        """Set eccentricity and mean_anomaly to zero."""
+        return_dict = {}
+        if tref_in is not None:
+            # check that tref_in is in the allowed range
+            ndim = np.ndim(tref_in)
```

Comment:
Is this code copied from somewhere else? If so, don't copy, make a function.

Reply:
Done!